src/utils/helpers.js

```diff
@@ -0,0 +1,30 @@
+export function SetButtonText(
```

Comment:
`SetButtonText`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)